openlane_update_tool/action.yml

```diff
@@ -0,0 +1,37 @@
+name: Update OpenLane Tool
+
+inputs:
+  tool_name:
+    description: Name of the tool to update.
+    required: true
+  pdk_root:
+    description: Path to the PDK folder.
+    default: .
+  build_ol:
+    description: Build docker image after tool update.
+    default: true
+
```

Comment:
should there be an input tag as well?

Reply:
Added on [d74b2bd](https://github.com/The-OpenROAD-Project/actions/pull/11/commits/d74b2bd9e00a47caec055071ecd2733318b4c9ae)